### PR TITLE
Add Wiki Maintainer Badge and Dropdown Item

### LINKED ISSUE #75: Add Wiki Maintainers badge and dropdown
Add Wiki Maintainers badge and dropdown along with 3da, rtps and so on

Group is `wiki_maintainers`

### PR DESCRIPTION
**Pull request by adamhb123**
resolves #75

Changes exemplified: 

badge-example
dropdown

### CODE REVIEW

File: profiles/static/style.css [hunk 1]
```diff
@@ -1,27 +1,27 @@
 .card-custom-wide-img {
-  height: 130px;
-  background-image: url("/static/assets/images/diamonds-light.png");
-  transition: all 0.5s ease;
+    height: 130px;
+    background-image: url("/static/assets/images/diamonds-light.png");
+    transition: all 0.5s ease;
```
Comment (mxmeinhold) [inline profiles/static/style.css:4]: Please don't reindent _everything_

**Review by mxmeinhold — CHANGES_REQUESTED** [review]

**Comment by adamhb123** [comment]
Examples of changes:
badge-example
dropdown

File: profiles/static/style.css (hunk 1 above)
Comment (adamhb123) [reply profiles/static/style.css]: Reverted changes to indentation, only changed originally to add uniformity.

**Review by mxmeinhold — APPROVED** [review]
LGTM! :ship: :it:

**Review by devinmatte — APPROVED** [review]
Good to go. Merging

**Pull request merged**